client/src/components/Common/ListHeader.vue

```diff
@@ -110,6 +114,8 @@ defineExpose({
 <style scoped lang="scss">
 .list-header {
     display: flex;
+    width: 100%;
+    padding: 0 0.5rem 0.5rem;
```

Comment:
Please avoid adding custom styles, we already have bootstrap, use d-flex, w-100, px or py.